### PR TITLE
[CS] Correction of translated words, typos, word order, etc.

### PR DESCRIPTION
**Pull request by I3elphegor**
A lot of words (but not all yet) have been changed to match the translation of the cards and the original rulebook (For example, Boj -> Souboj, Prohlášení Astrologů -> Hvězdářská Vyhláška, Vedlejší Hrdina -> Sekundární Hrdina, Životy -> HP, ...). Many words have been changed to be more consistent throughout the rules. While some typos have been squashed, I forgot to squash commits.

While I absolutely agree that some words are translated much, much better (see the four examples above) than in the original rulebook, those words are also found on the cards, city boards, etc. Such disparity can then lead to even a person quite familiar with the original rules of the game getting lost in the rulebook rewrite and subsequently the game itself.

### CODE REVIEW

File: translations/ai_rules.tex/cs.po
Comment (tomaas-zeman) [inline translations/ai_rules.tex/cs.po:210]:
```suggestion
"Pokud je AI Balíček během Souboje vyčerpán, přestaňte z něj dobírat.\n"
```

File: translations/ai_rules.tex/cs.po
Comment (tomaas-zeman) [inline translations/ai_rules.tex/cs.po:224]:
```suggestion
"\\subsection*{Souboj s AI}\n"
```

**Comment by tomaas-zeman** [comment]
The original translation sucks big time, nobody gave it a thought while doing it and they often selected completely wrong words ("Střílející jednotky"? Show me a game that uses that :D). They didn't even considered the original HOMM3 :/

But I agree it makes sense to be consistent with the terms on cards.

File: translations/ai_rules.tex/cs.po
Comment (tomaas-zeman) [inline translations/ai_rules.tex/cs.po:87]: I'm not sure about the "Souboj proti" - "Souboj s" is more natural

File: translations/all_map_locations.tex/cs.po
Comment (tomaas-zeman) [inline translations/all_map_locations.tex/cs.po:188]:
```suggestion
"  Efekty Polí, které vám umožňují \\textbf{Hledat} Kouzla nebo Artefakty nebo kde musíte utratit suroviny, abyste použili efekt Pole, jsou \\textbf{vždy volitelné}.\n"
```

File: translations/combat.tex/cs.po
Comment (tomaas-zeman) [inline translations/combat.tex/cs.po:131]:
```suggestion
"Pokud je Úroveň vašeho Hrdiny vyšší než Obtížnost Pole v Souboji proti Neutrálním Jednotkám, \\textbf{nedochází k žádnému Souboji}.\n"
```

File: translations/combat.tex/cs.po
Comment (tomaas-zeman) [inline translations/combat.tex/cs.po:530]: > Po zahrání všech karet pro útok je hozena Kostka Útoku
has different meaning
> Po zahrání všech karet je pro útok hozena Kostka Útoku

The original is
> After all cards for the attack have been played, the Attack Die is thrown

File: translations/components.tex/cs.po
Comment (tomaas-zeman) [inline translations/components.tex/cs.po:434]: Some terms in the "Component" section were intentionally shortened to fit. It now causes layout to overflow to next pages in printable version

https://github.com/Heegu-sama/Homm3BG/actions/runs/11044893987/artifacts/1980256486

Screenshot 2024-09-26 at 10 57 25

File: translations/map_elements.tex/cs.po
Comment (tomaas-zeman) [inline translations/map_elements.tex/cs.po:117]: I think it's important to mention that it has to be Visited right after the combat. I wouldn't change that part

File: translations/units.tex/cs.po
Comment (tomaas-zeman) [inline translations/units.tex/cs.po:209]:
```suggestion
"\\includegraphics[height=30px]{\\images/hp.png} \\textbf{\\pagetarget{HP}{HP}} -  Nebo-li životy. Hodnota poškození potřebná k poražení Jednotky.\n"
```

**Comment by tomaas-zeman** [comment]
Good job with this one! 🎉

File: translations/components.tex/cs.po
Comment (I3elphegor) [reply translations/components.tex/cs.po:434]: I thought it might happen. I'll install TexStudio and somehow try to get po4a working on Windows so I can do change checks and make sure it doesn't happen again.
And while I'm looking at the attached picture, I forgot to change "Posílená Karta Statistik" to "Vylepšená Karta" in Components.

File: translations/combat.tex/cs.po
Comment (I3elphegor) [reply translations/combat.tex/cs.po:530]: You are right.

File: translations/ai_rules.tex/cs.po
Comment (I3elphegor) [reply translations/ai_rules.tex/cs.po:87]: I'm not so sure about that either. It seems more natural, but it could also be misinterpreted to mean that if the AI heroes have neutral units with them, they automatically win the combat.

Comment (tomaas-zeman) [reply translations/ai_rules.tex/cs.po:87]: Hmm. I mean, it could but it would be a very strange rule.

We could either leave it here with "proti" or rephrase the sentence to something like "...Soubor s jakymikoliv Neutralnimi Jednotkami, na ktere narazi..." 🤷 

Unfortunately this "souboj proti" is in multiple places.

Comment (I3elphegor) [reply translations/ai_rules.tex/cs.po:87]: I would leave it with "proti".

**Comment by qwrtln** [comment]
You'll need to work on the layout, because the Czech digital document has 55 pages instead of 53. Probably needs some shortening.

File: translations/components.tex/cs.po
Comment (tomaas-zeman) [reply translations/components.tex/cs.po:434]: The components section still has broken layout. What used to be on 2 pages is now on 4 with lots of empty space. I suggest to keep the original words and only update those addressed by this PR, keeping them short

Comment (tomaas-zeman) [reply translations/components.tex/cs.po:434]: You can see tests below are failing because of that

Comment (I3elphegor) [reply translations/components.tex/cs.po:434]: I agree. Please, could you edit that? I still can't get po4a to work on Windows to view the rulebook, and check exactly what all needs to be changed back.

Comment (tomaas-zeman) [reply translations/components.tex/cs.po:434]: Im not sure how since your work on a fork.

@qwrtln Can we add @I3elphegor to maintainers here and recreate this PR here?

Comment (qwrtln) [reply translations/components.tex/cs.po:434]: I don't have such powers, but I can think of two easier solutions:
1. For @tomaas-zeman, since the following setting was ticked:
image
you can run `po4a`, and commit those changes yourself. See [instructions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork?platform=mac).
2. For @I3elphegor, you can open a PR in your fork, write a comment with text `po4a commit`, then the bot will run `po4a`, and commit changes to your PR. After that, you can merge the PR to the main branch in your fork, and those changes will be reflected here. See [example in another PR](https://github.com/Heegu-sama/Homm3BG/pull/442#issuecomment-2105990930), how this mechanism works.

Comment (I3elphegor) [reply translations/components.tex/cs.po:434]: I fixed the page count by adjusting the size of some tokens and dice in the Components section.

**Comment by I3elphegor** [comment]
I fixed the Components section in the Czech version, but I still need to shorten the Scholar and Witch's Hut in the All map locations section in the Czech version so that the rule book has the correct number of pages. I'll fix it tonight. I apologize for the mess I made here.

**Comment by DarthGandalf** [comment]
No worries!

**Review by tomaas-zeman — APPROVED** [review]

**Pull request merged**
